website/docs/r/schematics_agent.html.markdown

```diff
@@ -76,14 +68,23 @@ Nested scheme for **user_state**:
 	* `set_by` - (Computed, String) Name of the User who set the state of the Object.
 	* `state` - (Optional, String) User-defined states  * `enable`  Agent is enabled by the user.  * `disable` Agent is disbaled by the user.
 	  * Constraints: Allowable values are: `enable`, `disable`.
-* `version` - (Required, String) Agent version. Available Versions are 1.0.0-beta1, 1.0.0-beta2
+* `version` - (Required, String) Agent version. Available Versions are 1.0.0, 1.0.0-prega, 1.0.0-beta2, 1.0.0-beta1
```

Comment:
don't document these verison 1.0.0-prega, 1.0.0-beta2, 1.0.0-beta1
Instead of documenting since we need to change every time we release can we document to cloud docs where we will maintian list

Reply:
okay. will change